src/lib/components/composites/paper-components/ReadingListEntry.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+    import { goto } from "$app/navigation";
+    import { handleSingleOrDoubleClick } from "$lib/utils/common-helper";
+    import type { ReadingListEntryInterface } from "$lib/model/component-interfaces";
+    import PaperInfo from "$lib/components/composites/paper-components/PaperInfo.svelte";
+    import PaperBookmarkButton from "$lib/components/composites/PaperBookmarkButton.svelte";
+    import DownloadButton from "$lib/components/composites/DownloadButton.svelte";
+
+    type ReadingListEntryProps = ReadingListEntryInterface & {
+        onClick?: () => void;
+    };
+
+    const navigateToPaperView = () => {
+        goto(`/paper/${id}`);
+    };
+
+    const { paper, onClick = navigateToPaperView }: ReadingListEntryProps = $props();
+    const { id, ...paperWithoutId } = paper;
+
+    let timeoutId: ReturnType<typeof setTimeout> | null = $state(null);
+    const handleClick = () => {
+        timeoutId = handleSingleOrDoubleClick(timeoutId, onClick, navigateToPaperView);
+    };
+</script>
+
+<!--
+@component
+Container displaying the information of a paper on the reading list.
+
+This component shows the
+  - title of the paper
+  - authors (or 'unknown authors' if none are specified)
+
+Furthermore this component is clickable and navigates to the corresponding paper view,
+if the onClick() event handler is not overridden. Otherwise it executes the custom event handler
+on a single click. A double click always causes the navigation to the paper view.
+
+Usage:
+```svelte
+    <ReadingListEntry paper={paper} />
+```
+-->
+<div
+    class="border-container-border-grey highlight-on-hover flex w-full flex-row items-center gap-16 rounded-md border-b px-2.5 py-1.5"
```

Comment:
What about using a full border? I know we did this in the design for whatever reason, but I think it looks weird. Especially when the paper list entry looks entirely different.
```suggestion
    class="border-container-border-grey highlight-on-hover flex w-full flex-row items-center gap-16 rounded-md border px-2.5 py-1.5"
```

Reply:
For me it's the other way round: in my opinion it looks weird with a full border and I've tried it myself and I don't think it looks nice. It also gives you a visual distinction between `PaperListEntry` and `ReadingListEntry`.
I would suggest asking someone else, what do you think?

Reply:
haha
In my opinion, it's weird that there's a distinction, as they represent the same thing.
I started a poll in our team channel.

Reply:
Ok, I think the opinion of our team is clear: uniformity.